remote_report_to_printer/views/res_remote_views.xml

```diff
@@ -7,7 +7,7 @@
         <field name="arch" type="xml">
             <notebook position="inside">
                 <page name="printers" string="Printers">
-                    <field name="remote_printer_ids" />
+                    <field name="remote_printer_ids" colspan="2" />
```

Comment:
I think this does nothing. The other change has sense, but as we are inside a page without a group, we don't need to apply colspan here.

Reply:
Changes done! 👍🏽 Thx